pkg/audit/containment_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit_test
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/pkg/audit"
+	"storj.io/storj/pkg/pkcrypto"
+	"storj.io/storj/pkg/storj"
+)
+
+func TestContainIncrementAndGet(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 4,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+
+		randBytes := make([]byte, 10)
+		_, err := rand.Read(randBytes)
+		require.NoError(t, err)
```

Comment:
Is this a convention to also check errors from other packages in our test?

Reply:
I wouldn't say it's a convention but I need this function to create random test data and it returns an error, so I need to handle that error in the test. You will see `require.NoError(t, err)` used for this in other test files too.

Reply:
It isn't a matter of a convention is a matter of that a test can continue if the functions needed for the test return an error, so we have to abort the test if those happen and `require` do so.